packages/grid/_modules_/grid/components/toolbar/DensitySelector.tsx

```diff
@@ -25,17 +25,17 @@ export function DensitySelector() {
   const DensityOptions: Array<DensityOption> = [
     {
       icon: <DensityCompactIcon />,
-      label: 'Compact',
+      label: apiRef!.current.getText('densityOptionCompact') as string,
       value: DensityTypes.Compact,
     },
     {
       icon: <DensityStandardIcon />,
-      label: 'Standard',
+      label: apiRef!.current.getText('densityOptionStandard') as string,
```

Comment:
Do we need a getter? Why not access the prop directly?

Reply:
I would say yes - the reason is to be aligned with the general approach of implementing features in the grid.